tools/scripts/build/templateInstallationPackage/scriptPreffix.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
Is this licence header valid for sh files?
I think that it should be like the _tools/scripts/build/templateInstallationPackage/unistall.sh_ file.